File: packages/coterie-py/examples/01_single_agent.py
"""Minimal smoke test: build the graph and dry-run without spawning the real CLI.

Run with: python examples/01_single_agent.py
"""

from coterie.graph import build_graph


def main() -> None:
    graph = build_graph(agent_id="claude", adapter_kind="claude-code", workdir=".")
    print("graph compiled:", graph)
    print("nodes:", list(graph.get_graph().nodes))


if __name__ == "__main__":
    main()

File: packages/coterie-js/examples/01_single_agent.ts
/**
 * Minimal smoke check: build the graph without invoking the real CLI.
 * Run with: npx tsx examples/01_single_agent.ts
 */
import { buildGraph } from "../src/graph.js";

const graph = buildGraph({ agentId: "claude", adapterKind: "claude-code", workdir: "." });
console.log("graph compiled:", graph);
